### ISSUE TITLE
Inconsistent metrics configuration across components

### ISSUE THREAD

**Issue by adrianmo** [pravega/pravega#3059]
**Problem description**
Metrics configuration is not consistent across Pravega components.

As an example, if we want to override the StatsD host to send our metrics, on the Segmentstore we would need to use the `metrics.statsDHost`:

```
metrics.statsDHost=nautilus.pravega.telegraf
```

Whereas on the Controller, the key to update is `config.controller.metricstatsDHost`:

```
config.controller.metricstatsDHost=nautilus.pravega.telegraf
```

This has already caused confusion and issues to users. For example, the Pravega operator does not differentiate between Controller and Segmentstore configuration, so in order for the metrics configuration to work in all components, the same option needs to be set multiple times.

**Problem location**
Configuration files and classes.

**Suggestions for an improvement**
The Controller expects metrics configuration to have the prefix `config.controller.metric` ([source](https://github.com/pravega/pravega/blob/v0.4.0-rc0/controller/src/main/java/io/pravega/controller/util/Config.java#L102)). However, according to the [config file](https://github.com/pravega/pravega/blob/v0.4.0-rc0/config/config.properties#L155), the prefix should be just `metrics.` (notice the final dot, not present in the current Controller prefix).

**Comment by andreipaduroiu** [comment]
This should be fixed by #489.

**Comment by RaulGracia** [comment]
Configuration has been standardized as part of #3406.
